.eslintrc

```diff
@@ -39,7 +40,20 @@
     "dot-notation": [2, {"allowKeywords": true}],
     "eqeqeq": [2, "allow-null"],
     "guard-for-in": 0,
-    "indent": [2, 2],
+    "indent": [2, 2, {
+      "SwitchCase": 1,
+      "VariableDeclarator": 1,
+      "outerIIFEBody": 1,
+      "MemberExpression": 1,
+      "FunctionDeclaration": {
+        "parameters": 1,
+        "body": 1
+      },
+      "FunctionExpression": {
+        "parameters": 1,
+        "body": 1
+      }
+    }],
     "keyword-spacing": 2,
```

Comment:
An additional rule that we can add here is ["key-spacing"](http://eslint.org/docs/rules/key-spacing), I tried to add it in our config (without changing anything in the rule default behavior) and it caught a couple of spacing errors in `src/firefox/preferences.js` and `src/program.js`.